### DISCUSSION TITLE
Msg doesn't reach topic

### DISCUSSION THREAD

**Discussion by supermom5280** [strimzi/strimzi-kafka-operator#9382] [General]
Strimzi 36.1 and kafka 3.5 .. mTLS is turned on between kafka & clients.  The client is running within a container within the same kubernetes cluster as strimzi but within a different namespace.  The client has ACLs set for it as a producer and a creator.   When the client starts up it's able to create the producer,  the topic within the Strimzi cluster and appears to send the message successfully.  There are no errors on the client side or the server side and NO SSL error either but when looking that the topic on the kafka server the message never appears in <topicname>/00000000000000.log.   I have an external client using mTLS that is configured for the same cluster and can successfully write and create a topic  what am i missing?

**Comment by supermom5280** [comment]
more information.  Use our own self signed certs.  Same different CA for the internal as the external.  Both are in the client-ca-cert the same way. Though the internal client does seem to authenticate fully without error.

**Reply by scholzj** [reply]
I think you will also need to share your configurations, logs, etc.
